mist-api/src/routes/genes/post.js

```diff
@@ -1,10 +1,15 @@
 'use strict'
 
-const getPostHelpers = require('./get-post-helpers')
+// Vendor
+const bodyParser = require('body-parser')
+// Local
+const { geneFinderMiddlewares, docs } = require('./genes-route-helpers');
 
-module.exports = function(app, middlewares, routeMiddlewares) {
-  const isPOST = true
-  return getPostHelpers.geneFinder(app, middlewares, isPOST)
+module.exports = function(app, middlewares) {
+  return [
+    bodyParser.urlencoded({extended: false}),
+    ...geneFinderMiddlewares(app, middlewares, (req) => { return req.body })
```

Comment:
Non-blocking, but a slightly more concise way to write the `inputGetter` is: `(req) => req.body`